src/App.js

```diff
@@ -14,6 +14,7 @@ function App() {
       <Routes>
         <Route path="/" element={<Books />} />
         <Route path="/Category" element={<Category />} />
```

Comment:
- [ ]  When I open` /Category `path  I can see an empty page, this is happening because you are missing   `return ()` keyword to return JSX in `Category.js` file.  Please make sure to make the page content is properly displayed